webapp/src/monaco.tsx

```diff
@@ -913,6 +898,36 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         this.editor.focus();
     }, 200);
 
+    protected onPointerMove = (ev: any) => {
+        if (this.insertionSnippet) {
+            this.dragCurrentPos = {
+                x: ev.clientX,
+                y: ev.clientY
+            }
+            this.onDragBlockThrottled(ev);
+        }
+    };
+
+    protected onPointerUp = (ev: any) => {
```

Comment:
these `any`s could be `MouseEvent`, right?

Reply:
sure, the touch event is different so it'd have to be MouseEvent | TouchEvent but that's probably still better than any